util/dvsim/LocalLauncher.py

```diff
@@ -47,9 +46,9 @@ def _do_launch(self):
                      "w",
                      encoding="UTF-8",
                      errors="surrogateescape")
-            f.write("[Executing]:\n{}\n\n".format(self.deploy.cmd))
+            f.write("[Executing]:\n{}\n\n".format(" ".join(self.deploy.cmd)))
```

Comment:
To be helpful when debugging, it might be better if this was `" ".join(shlex.quote(arg) for arg in self.deploy.cmd)`